go.mod

```diff
@@ -230,12 +230,11 @@ require (
 	go.uber.org/atomic v1.9.0 // indirect
```

Comment:
Have we merged from `main`? I reckon these shouldn't be here if we already bumped the dependencies.